ettienekorayyi/poc-code-sydney
# Added callback function on the MentorState.js to allow us to pass the…

… filteredMentors to the result component.

## Rationale

Refactored the search functionality to enable us to pass the filtered mentors to the results component.

## Advice for Reviewers & Testing Notes

- Run the App
- On the page the list of mentors should be displayed and the search results should be hidden
- Click the search box, type dorathy then click Enter should display the card component with 1 search result.
- Click the dropdown, choose title then enter developer, this should display 10 search results.

## Screenshots:

- Add a screenshot or two of your changes, where applicable.
![image](https://user-images.githubusercontent.com/19362078/80000735-dc419080-8500-11ea-9f86-22aae5e2cea1.png)

![image](https://user-images.githubusercontent.com/19362078/80000781-ed8a9d00-8500-11ea-849d-5ac8136d8cb9.png)

![image](https://user-images.githubusercontent.com/19362078/80000861-05fab780-8501-11ea-9b01-6854176ed9b9.png)



## Task Name

- Task name from Trello
DEV-019 Search Functionality - Search By Name, Title

## Linting Checklist
